src/Stratis.Bitcoin/Consensus/PeerGetHeaderRequestBehavior.cs

```diff
@@ -39,7 +39,7 @@ public sealed class RateLimitingBehavior : NetworkPeerBehavior
         private int getHeaderRequestCount;
 
         /// <summary>The threshold after which the node will be banned and disconnected.</summary>
-        private const int GetHeaderRequestCountThreshold = 10;
+        private const int GetHeaderRequestCountThreshold = 50;
```

Comment:
mmm I thought already that 10 was too much, why updating it even to 50?
what's a valid scenario that allow us to reply to almost 50 request of the same blocklocator in a minute?

If there is a condition that's valid, then just not consider it (e.g. if it's valid to have 50 request of getheaders requiring blocks from current tip) but if you allow to request 50 getheaders you are just mitigating the issue

and a consideration: is the timeframe needed?

Reply:
as this is a temporary solution we its ok to keep the time frame